controllers/performanceprofile_controller.go

```diff
@@ -278,6 +278,14 @@ func (r *PerformanceProfileReconciler) Reconcile(req ctrl.Request) (ctrl.Result,
 		}
 	}
 
+	// get tuned profile degraded conditions
+	if conditions == nil {
+		conditions, err = r.getTunedConditionsByProfile(instance)
+		if err != nil {
+			return r.updateDegradedCondition(instance, conditionFailedGettingMCPStatus, err)
```

Comment:
You should change `conditionFailedGettingMCPStatus` to something like
`conditionFailedGettingTunedProfileStatus`